things/postgres/channels.go

```diff
@@ -438,8 +448,18 @@ func getMetadataQuery(m things.Metadata) ([]byte, string, error) {
 	return mb, mq, nil
 }
 
-type dbConnection struct {
-	Channel string `db:"channel"`
-	Thing   string `db:"thing"`
-	Owner   string `db:"owner"`
+func (cr channelRepository) total(ctx context.Context, query string, params map[string]interface{}) (uint64, error) {
```

Comment:
Can you create single helper function for this? (maybe this method can receive `db` as parameter as well)